### PR TITLE
Visitors get access to read Backyard Conspiracies

### PR DESCRIPTION
**Pull request by arunbhalli**
[**PT-Story:**  ](https://www.pivotaltracker.com/story/show/176030840)
### User Story 
``` 
As a Visitor
To see what's going on in my neighborhood
I want to see to Backyard Conspiracies
``` 
## Changes proposed in this pull request: 
* Menu tab that links to Backyard Conspiracy page
* Makes use of geolocation to showcase localized backyard articles
* Displaying backyard articles in a list form
## What we have learned:
* Deeper knowledge in redux
* Using IDs in routes

## Screenshots
https://recordit.co/HQSwwBUxpY

image

image

### CODE REVIEW

File: src/state/store/initialState.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
 const initialState = {
   error: false,
   articles: [],
+  backyardArticles: [],
   article: {},
   subscriber: false,
   open: false,
+  backyardArticle: [],
```
Comment (lukeperrin10) [inline src/state/store/initialState.js]: Shouldn't a single backyard article be an object like article?

Comment (christerforsgren91) [reply src/state/store/initialState.js]: yes, its fixed now!

File: src/components/BackyardDashboard.jsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                : 'Allow your location!'}
+            </h2>
+          </Segment>
+          <Table celled padded inverted style={{ overflowY: 'scroll' }}>
+            <Table.Header>
+              <Table.Row textAlign='center'>
+                <Table.HeaderCell singleLine>Title</Table.HeaderCell>
+                <Table.HeaderCell>Theme</Table.HeaderCell>
+                <Table.HeaderCell>Written by</Table.HeaderCell>
+                <Table.HeaderCell>Updated On</Table.HeaderCell>
```
Comment (offcast-dev) [inline src/components/BackyardDashboard.jsx]: I think we don't update Backyard Articles, so you can remove it

File: src/components/layout/BackyardArticle.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+import BackyardArticles from '../../modules/BackyardArticles';
+import { useParams } from 'react-router';
+
+const BackyardArticle = () => {
+  const { backyardArticle } = useSelector((state) => state);
+  const { id } = useParams();
+
+  useEffect(() => {
+    BackyardArticles.show(id);
+    // eslint-disable-next-line
```
Comment (offcast-dev) [inline src/components/layout/BackyardArticle.jsx:12]: remove

**Review by offcast-dev — CHANGES_REQUESTED** [review]
Remove couple of things. Also check the PR description, you gave extra star and 'more Redux' is not very descriptive. Also this PR could've been split into 2: show list of BArticles and show individual BArticle.

File: src/components/layout/BackyardArticle.jsx (hunk 3 above)
Comment (1CIM) [reply src/components/layout/BackyardArticle.jsx:12]: That line removes the BS warning on the line after

File: src/components/BackyardDashboard.jsx (hunk 2 above)
Comment (christerforsgren91) [reply src/components/BackyardDashboard.jsx]: done

File: src/components/layout/BackyardArticle.jsx (hunk 3 above)
Comment (offcast-dev) [reply src/components/layout/BackyardArticle.jsx:12]: interesting..

**Review by offcast-dev — APPROVED** [review]

**Pull request merged**